assistant/voice_assistant.py

```
@@ -0,0 +1,405 @@
+import numpy as np
+import torch
```

Ruff (0.12.2)

`torch` imported but unused

Remove unused import: `torch`

(F401)